f() -/>
  x <- g()
    catch err: err
